Thanks @vojtechjelinek, LGTM!

FYI one backend test failure: 

```
======================================================================
FAIL: test_cannot_access_exploration_with_incorrect_schema (core.controllers.base_test.SchemaValidationUrlArgsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/oppia/oppia/core/controllers/base_test.py", line 1612, in test_cannot_access_exploration_with_incorrect_schema
    self.assertEqual(response['error'], error_msg)
AssertionError: "At 'http://localhost/mock_play_exploratio[118 chars]p_id" != "Schema validation for 'exploration_id' fa[38 chars]p_id"
- At 'http://localhost/mock_play_exploration/exp_id' these errors are happening:
  Schema validation for 'exploration_id' failed: Could not convert str to int: exp_id
 
```

--- a/core/controllers/base_test.py
+++ b/core/controllers/base_test.py
@@ -1607,6 +1607,8 @@ class SchemaValidationUrlArgsTests(test_utils.GenericTestBase):
                 '/mock_play_exploration/%s' % self.exp_id,
                     expected_status_int=400)
             error_msg = (
+                'At \'http://localhost/mock_play_exploration/exp_id\' '
+                'these errors are happening:\n'
                 'Schema validation for \'exploration_id\' failed: Could not '
                 'convert str to int: %s' % self.exp_id)
             self.assertEqual(response['error'], error_msg)

